tests/queries/tests.py

```diff
@@ -1168,6 +1168,48 @@ def test_ticket_20250(self):
             ['<Author: a1>', '<Author: a2>', '<Author: a3>', '<Author: a4>']
         )
 
+    def test_delete_returns_num_rows(self):
```

Comment:
Don't think this is the best app for the tests. tests/delete seems like a good option, at least for the queryset tests.